contracts/ReputationMiningCycle.sol

```diff
@@ -15,7 +15,7 @@
   along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
 */
 
-pragma solidity >=0.5.3;
+pragma solidity 0.5.8;
 pragma experimental "ABIEncoderV2";
 
```

Comment:
Note to self, when checking coverage I noticed we're not testing the `colony-reputation-index-beyond-reputation-log-length` error. Should be simple to add, maybe in this PR.

Reply:
Ah no we can't because it's a `view` function..